crm_request/models/request.py

```diff
@@ -177,6 +180,20 @@ def message_new(self, msg, custom_values=None):
             request.language = self.detect_lang(
                 request.description).lang_id.code
 
+        # Check here if the date of the mail is during a holiday
+        mail_date = fields.Date.from_string(msg.get('date'))
```

Comment:
You can simply use the string format (in condition that the string is formatted correctly yyyy-mm-dd), because the search method takes string as values and not Date objects.
```suggestion
        mail_date = msg.get('date')
```